src/main/java/mate.academy/service/impl/FileReaderServiceImpl.java

```diff
@@ -4,8 +4,10 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.service.FileReaderService;
 
+@Component
```

Comment:
Please make sure to import the @Component annotation from the correct package defined in the task description.